content_sync/pipelines/definitions/concourse/site_pipeline_test.py

```diff
@@ -427,10 +427,12 @@ def assert_base_build_tasks(tasks: list[dict], offline: bool):  # noqa: FBT001
                 open_discussions_webhook_step_online_params["version"]
                 == config.vars["pipeline_name"]
             )
+    offline_build_gate_put_task = online_site_tasks[-1]
     assert (
-        online_site_tasks[-1]["try"]["put"]
+        offline_build_gate_put_task["try"]["put"]
         == pipeline_definition._offline_build_gate_identifier  # noqa: SLF001
     )
+    assert online_site_tasks[-1]["try"]["no_get"] is True
```

Comment:
a nitpick, but maybe you wanted this here:
```suggestion
    assert offline_build_gate_put_task["try"]["no_get"] is True
```